src/storage/src/hummock/sstable/forward_sstable_iterator.rs

```diff
@@ -162,7 +162,7 @@ impl SstableIterator {
             let next_to_start_idx = start_idx + 1;
             if next_to_start_idx <= self.read_block_meta_range.1 {
                 let end_idx = match bound {
-                    Unbounded => self.read_block_meta_range.1,
+                    Unbounded => self.read_block_meta_range.1 + 1,
```

Comment:
nits: to avoid confusion and future bugs related to `read_block_meta_range`, I think you can make the naming clearing:
```
- read_block_meta_range: (usize, usize),
+ block_start_idx_inclusive: usize,
+ block_end_idx_inclusive: usize
```

Reply:
Thanks, done